### PR TITLE
Enhance/101 responsive update for the footer

### PR DESCRIPTION
**Pull request by LULIU-9**
### Descriptions

- Link to UI design (if possible):
figma.com/file/q2ZHM8s0LcNxz3otGFLDzO/Website?node-id=0%3A1&t=iAOMerc7vttUgFmz-0
### Provide screenshots or videos that the function is working.

https://user-images.githubusercontent.com/69507783/225546010-99571ce4-8c49-432e-980d-cb11d81f632a.mov

### Provide screenshots of no warning, or error in the console.
Screen Shot 2023-03-16 at 5 33 37 pm

- [x] All unit tests passing and coverage of testing is more than 90% unless special needs.
- [x] No merge conflict with the `main` branch, make sure your code is up to date with `main`.
- [x] Check PR title is aligned with your ticket.
- [x] Check your app will run without crashes.
- [x] Check if there are hard-coded values.
- [x] No `//eslint-disable-next-line no-unused-vars`, unless otherwise specified.
- [x] Check you are using `camelCase && PascalCase` correctly.
- [x] Remove unused comments.

### CODE REVIEW

File: styles/Footer.module.css [hunk 1]
```diff
@@ -96,3 +96,54 @@
...
+  .footer_top_link__ul {
+    align-items: center;
+  }
+  .footer_bottom__container {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+  }
+  .footer_bottom__privacy {
+    position: static;
```
Comment (yunshen513) [inline styles/Footer.module.css:142]: Looks good, but may not necessarily position these elements using "absolute" before and "static" here. Would putting them in a flexbox and use "space-between" an easier way?

Comment (LULIU-9) [reply styles/Footer.module.css]: Thanks

**Review by Trent1900 — APPROVED** [review]
LGTM

**Review by tatoMa — APPROVED** [review]
LGTM

**Pull request merged**